stack/lab/before-notebook.d/60_prepare-aiidalab.sh

```diff
@@ -6,11 +6,6 @@ set -x
 # Environment.
 export SHELL=/bin/bash
 
-# Fix https://github.com/aiidalab/aiidalab-docker-stack/issues/225
```

Comment:
This issue has been fixed a long time ago and should not be relevant in the new docker stack.